### PR TITLE
 Remove hessian required minimizers for scipy default #1114

### LINKED ISSUE #1114: Remove hessian required minimizers from the default scipy minimizers
Currently when running scipy software it runs with the hessian required minimzers. A default scip run doesn't have the hessian enabled.  

image

- [ ] remove hessian solvers from default minimizers

### PR DESCRIPTION
**Pull request by joshuadkitenge**
#### Description of Work

Fixes #1114

#### Testing Instructions

1.
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?

### CODE REVIEW

**Review by jess-farmer — COMMENTED** [review]
Just have one comment - I think it would be good to add a note to the minimizer options page in the docs to let users know that hessian required minimizers have to be set explicitly in order to be run

**Comment by joshuadkitenge** [comment]
Some of the documentation had incorrect formatting. I've fixed that in the latest commit.

File: docs/source/users/options/minimizer_option.rst [hunk 1]
```diff
@@ -312,6 +314,10 @@ spectrometers.*  We currently support:
 `proprietary product <https://www.mathworks.com/pricing-licensing.html>`__.
 Horace is made available under the the `GPL-3 licence <https://www.gnu.org/licenses/gpl-3.0.html>`__.
 
+.. code-block:: rst
```
Comment (jess-farmer) [inline docs/source/users/options/minimizer_option.rst:317]:
```suggestion
.. code-block:: rst

```
There should be a blank line here for the code-block to work

File: docs/source/users/options/minimizer_option.rst [hunk 2]
```diff
@@ -672,6 +678,10 @@ The SciPy minimizers are set as follows:
            trust-constr
            dogleg
 
+.. note::
+   The Hessian enabled solvers are not run by default when `scipy` software is selected. In order to run this minimizer, you must explicitly
+   set it as above.
```
Comment (jess-farmer) [inline docs/source/users/options/minimizer_option.rst:683]:
```suggestion
   The Hessian enabled solvers are not run by default when `scipy` software is selected. In order to run these minimizers, you must explicitly
   set them as above.
```

**Review by jess-farmer — CHANGES_REQUESTED** [review]
Just two very minor changes, otherwise looks good!

**Review by jess-farmer — APPROVED** [review]

**Pull request merged**